app/styles/home.scss

```diff
@@ -56,9 +56,10 @@ home {
       margin-bottom: 0;
     }
 
-    button {
+    .btn {
       display: inline-block;
       width: 180px;
+      padding-top: 14px;
```

Comment:
bahahha. instead of this here, can you remove `height: 43px;` and add `padding: 15px;` to [here](https://github.com/robinjoseph08/pokedextracker.com/blob/be1ad1a6d69483343c2847175a72ecef2268d639/app/styles/main.scss#L94)?